docs/docs-content/palette-cli/commands/fips-validate.md

```diff
@@ -0,0 +1,341 @@
+| `-n`           | `--namespace` | The namespace in which to deploy the resources for the FIPS scan. The default namespace is `vertex-ns` | string   |
+| `-f`           | `--out`       | The output file path to save the scan results. The default output is the terminal.                     | string   |
+| `-h`           | `--help`      | Display the help message for the `images` subcommand.                                                  | boolean  |
+
+### Limitations
+
+- Images hosted in private registries are not supported.
+- Only Go binaries are supported. Images with binaries compiled in other languages are marked as `unknown` in the
+  report.
+
+### Examples
+
+Validate the FIPS compliance of the images in your cluster.
+
+```shell
+palette fips-validate images
+```
+
+Validate the FIPS compliance of the images in your cluster and save the results to a file.
+
+```shell
+palette fips-validate images --out /path/to/fips-scan-results.txt
+```
+
+Validate the FIPS compliance of the images in your cluster and deploy the resources in a custom namespace.
+
+```shell
+palette fips-validate images --namespace my-scan-ns
+```
+
+### Review Results
+
+The report contains a list of images and the FIPS compliance status of the binaries in the images. Each row in the
+report contains the following columns:
+
+| **Column** | **Description**                                                                                                                           |
+| ---------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `NO.`      | The number of the image in the order of the scan.                                                                                         |
+| `BINARY`   | The name of the executable binary.                                                                                                        |
+| `ARCH`     | The architecture of the binary.                                                                                                           |
+| `PATH`     | The path of the binary in the image.                                                                                                      |
+| `VERSION`  | The version of the programming language used to create the binary. Go is the only supported language from a version detection capability. |
+| `EXE`      | How the binary was compiled, either statically or dynamically.                                                                            |
+| `CRYPTO`   | The cryptographic library used to compile the binary.                                                                                     |
+
+The following is an example of the FIPS compliance report. The following image is FIPS compliant as the binary is
+compiled with the BoringSSL cryptographic library.
```

Comment:
Is this the only check it does? Asking as I don't see a similar checks table as under `services`

Reply:
Yup, that's it